docs/source/configuration/multinode_cassandra.rst

```diff
@@ -0,0 +1,112 @@
+
+A quorum (but not all) of nodes are up
+======================================
+
+The following behaviour is guaranteed when interacting with a three node Cassandra cluster with one node down. More generally, this is the expected behaviour for a cluster that has one node down, but still has quorum.
+
+.. list-table::
+    :widths: 40 40
+    :header-rows: 1
+
+    *    - Cassandra KVS API Method
+         - Behaviour
+
+    *    - ``addGarbageCollectionSentinelValues``
+         - Same as when all nodes are up.
+
+    *    - ``cleanUpSchemaMutationLockTablesState``
+         - Same as when all nodes are up.
+
+    *    - ``close``
+         - Same as when all nodes are up.
+
+    *    - ``compactInternally``
+         - Same as when all nodes are up.
+
+    *    - ``create``
+         - Same as when all nodes are up.
+
+    *    - ``createTable``
+         - Throws ``IllegalStateException``
+
+    *    - ``createTables``
+         - Throws ``IllegalStateException``
+
+    *    - ``delete``
+         - Throws ``IllegalStateException``
+
+    *    - ``dropTable``
+         - Throws ``IllegalStateException``
+
+    *    - ``dropTables``
+         - Throws ``IllegalStateException``
+
+    *    - ``get``
+         - Same as when all nodes are up.
+
+    *    - ``getAllTableNames``
+         - Same as when all nodes are up.
+
+    *    - ``getAllTimestamps``
+         - Same as when all nodes are up.
+
+    *    - ``getLatestTimestamps``
+         - Same as when all nodes are up.
+
+    *    - ``getMetadataForTable``
+         - Same as when all nodes are up.
+
+    *    - ``getMetadataForTables``
+         - Same as when all nodes are up.
+
+    *    - ``getRange``
+         - Same as when all nodes are up.
+
+    *    - ``getRangeOfTimestamps``
+         - Same as when all nodes are up.
+
+    *    - ``getRows``
+         - Same as when all nodes are up.
+
+    *    - ``getRowsColumnRange``
+         - Same as when all nodes are up.
+
+    *    - ``multiPut``
+         - Same as when all nodes are up.
+
+    *    - ``put``
+         - Same as when all nodes are up.
+
+    *    - ``putMetadataForTable``
+         - Throws ``IllegalStateException`` after 1 minute.
+
+    *    - ``putMetadataForTables``
+         - Throws ``IllegalStateException`` after 1 minute.
+
+    *    - ``putUnlessExists``
+         - Same as when all nodes are up.
+
+    *    - ``putWithTimestamps``
+         - Same as when all nodes are up.
+
+    *    - ``truncateTable``
+         - Throws ``IllegalStateException``
+
+    *    - ``truncateTables``
+         - Throws ``IllegalStateException``
+
+Less than a quorum of nodes are up
+==================================
+
+Authentication will fail with a ``AuthenticationException``, since quorum is necessary for this operation. It is therefore guaranteed that the cluster is inaccessible in this case.
```

Comment:
@gmaretic @jboreiko Check out https://docs.datastax.com/en/datastax_enterprise/4.5/datastax_enterprise/sec/secConfSysAuthKeyspRepl.html#secConfSysAuthKeyspRepl__secReplFac

So the Cassandra superuser (which is what we use) requires QUORUM but other (non-super) users only require LOCAL_ONE. We should probably probably reword this part?